build.sbt

```diff
@@ -10,7 +10,7 @@ scalacOptions ++= List(
   "-unchecked",
   "-deprecation",
   "-language:_",
-  "-target:jvm-1.7",
+  "-target:jvm-1.6",
```

Comment:
Yes, it would be nice to have it published for Java 6, so the project can at least load on a Java 6 CI environment even if you don't run the jmh tests there :wink: